main/templates/base/form_buttons.html

```diff
@@ -3,35 +3,27 @@
 <div class="mt-3 mb-3 form-buttons">
     <div class="button-grid">
         {% if not no_back %}
-            <input class="button"
+            <input class="button mr-auto button-black"
                    type="submit"
                    name="save_back"
-                   value="{% trans '<< Opslaan en vorige stap' %}" />
-            {% if proposal %}
-                {% if proposal.is_pre_assessment %}
-                    <a class="button js-submit-redirect"
-                       href="{% url 'proposals:update_pre' proposal.pk %}">{% trans "Terug naar begin aanvraag" %}</a>
-                {% elif proposal.is_practice %}
-                    <a class="button js-submit-redirect"
-                       href="{% url 'proposals:update_practice' proposal.pk %}">
-                        {% trans "Terug naar begin aanvraag" %}
-                    </a>
-                {% else %}
-                    <a class="button js-submit-redirect"
-                       href="{% url 'proposals:update' proposal.pk %}">{% trans "Terug naar begin aanvraag" %}</a>
-                {% endif %}
+                   value="{% trans '<< Vorige stap' %}" />
+            {% if create_session %}
+                {% trans 'Maak een nieuwe sessie aan >>' as create_session_text %}
+                <input class="button button-colored continue-button"
+                       type="submit"
+                       name="create_new_session"
+                       value="{{ next_text|default:create_session_text }}" />
             {% endif %}
-            {% if study %}
-                <a class="button js-submit-redirect"
-                   href="{% url 'studies:update' study.pk %}">{% trans "Terug naar begin traject" %}</a>
-            {% endif %}
-            {% if session %}
-                <a class="button js-submit-redirect"
-                   href="{% url 'tasks:start' session.pk %}">{% trans "Terug naar begin sessie" %}</a>
+            {% if task %}
+                {% trans 'Maak een nieuwe taak aan >>' as create_task_text %}
+                <input class="button button-colored continue-button"
+                       type="submit"
+                       name="create_new_task"
+                       value="{{ next_text|default:create_task_text }}" />
             {% endif %}
         {% endif %}
```

Comment:
> The create task/session buttons are currently scoped within the {% if not no_back %} if statement. That doesn't cause any problems right now but I don't think it's right.

Yeah, good spot. It indeed does not cause any problems right now (if ever), but it's is kinda weird conceptually...

> Maybe it's always been that way but it can be fixed with an else clause and empty div:

Nope, I introduced this bug in this PR. I've pushed a fix for it, which also moves the new buttons to the right scope. (As it made the fix easier and more readable as well).